fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/binders/FilterBinders.java

```diff
@@ -28,13 +43,87 @@
 import javax.validation.constraints.NotNull;
 
 /**
- * Factory for ApiFilter objects. Has methods to build an ApiFilter object out of the filter clause in a Fili Api
- * request, and producing an ApiFilter by unioning the values of two provided ApiFilters.
+ * This utility class captures default implementations for binding and validating API models for filtering requests.
  */
-public class ApiFilterGenerator {
-    private static final Logger LOG = LoggerFactory.getLogger(ApiFilterGenerator.class);
+public class FilterBinders {
```

Comment:
This rename is dangerous (for example, people may be using the old class to
generate filters). The old class should be deprecated, instead of removed
completely.

Reply:
Darn.  I let Ben merge at that location, knowing I'd want to move it in the very next check in.  I hadn't counted on people rebasing in between.